# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/pyrequests/setup.patch
# Copyright (C) 2022 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- requests-2.28.0/setup.py.orig	2022-06-24 20:22:12.526802052 +0200
+++ requests-2.28.0/setup.py	2022-06-24 20:22:28.019801965 +0200
@@ -95,7 +95,6 @@
     package_dir={"": "src"},
     include_package_data=True,
     python_requires=">=3.8",
-    install_requires=requires,
     license=about["__license__"],
     zip_safe=False,
     classifiers=[
